cpp/ql/lib/semmlecode.cpp.dbscheme.stats

```diff
@@ -2634,2445 +2395,6 @@
 <dependencies/>
 </relation>
 <relation>
```

Comment:
I think you want to delete this line, but keep the last `<relation>` on line 5057? Currently it doesn't look like well-bracketed XML to me.